resources/sql_scripts/update_scripts/rsus_snmp_column_update.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE public.rsus 
+    RENAME COLUMN snmp_version_id TO snmp_protocol_id
+
+ALTER TABLE public.rsus     
+    DROP CONSTRAINT fk_snmp_version_id
```

Comment:
Will renaming the column work if it is done before dropping the foreign key constraint?

Reply:
I've reordered this so that the foreign key constraint is dropped before the column rename